libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/LocaleUtils.kt

```diff
@@ -0,0 +1,84 @@
+package com.mapbox.services.android.navigation.v5.utils
+
+import android.content.Context
+import android.os.Build
+import com.mapbox.api.directions.v5.DirectionsCriteria
+import java.util.Locale
+
+class LocaleUtils {
```

Comment:
ptal at https://github.com/mapbox/mapbox-navigation-android/pull/2189
I removed this dependency at all 😞

Reply:
I already did and discussed with Pablo. It would be much easier, to merge this and when you rebase, just delete this file